app/templates/components/ui-table/cell/cell-roles.hbs

```diff
@@ -1 +1 @@
-<span>sample@gmail.com (Organizer)</span>
+<span>{{record.roles}}({{t 'Organizer'}})</span>
```

Comment:
here organizer in not dummy data it is hard coded here should I remove it as I think it will be passed along with record.roles